src/shipchain_common/pagination.py

```diff
@@ -14,10 +14,18 @@
 limitations under the License.
 """
 
-from rest_framework_json_api.pagination import PageNumberPagination
+from rest_framework_json_api.pagination import JsonApiPageNumberPagination
 
 
-class CustomResponsePagination(PageNumberPagination):
+class JsonResponsePagination(JsonApiPageNumberPagination):
```

Comment:
We lost some fidelity on what this class does with the rename to `JsonResponsePagination`. How about `JsonApiPagePagination`?

Reply:
`JsonApiPagePagination` sound good to me :+1: